R_work/include/util_plot.R

```diff
@@ -1428,10 +1428,10 @@ calcSeriesSums = function( d , strPerm, lineTypes, intPerm=1,  T=c(), oy_total=c
         # table-based stat calculations
 
         dfSeriescustom_sums = list(
-          reliability = custom_sum(dfPctPerm$n_i * (dfPctPerm$thresh - dfPctPerm$o_bar_i)^2) / T[intPerm],
-          resolution = custom_sum(dfPctPerm$n_i * (dfPctPerm$o_bar_i - o_bar[intPerm])^2) / T[intPerm],
-          uncertainty = o_bar[intPerm] * (1 - o_bar[intPerm]),
-          baser = o_bar[intPerm],
+          reliability = custom_sum(dfPctPerm$n_i * (dfPctPerm$thresh - dfPctPerm$o_bar_i)^2) / T_table,
+          resolution = custom_sum(dfPctPerm$n_i * (dfPctPerm$o_bar_i - o_bar[intPerm])^2) / T_table,
+          uncertainty = o_bar_table * (1 - o_bar_table),
+          baser = o_bar_table,
```

Comment:
In MET, baser() is computed as "event_col_total()/n()":
https://github.com/dtcenter/MET/blob/6b951f395d35b81f28cea8aa150734abdc600866/met/src/libcode/vx_statistics/contable_nx2.cc#L471
And that's exactly the same computation for obar():
https://github.com/dtcenter/MET/blob/6b951f395d35b81f28cea8aa150734abdc600866/met/src/libcode/vx_statistics/contable_nx2.cc#L434